workers/typescript/src/activities.ts

```diff
@@ -0,0 +1,7 @@
+export async function noop() {
+  return undefined;
+}
+
+export async function delay(delay_for_ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, delay_for_ms));
+}
```

Comment:
```suggestion
export { sleep as delay } from '@temporalio/activity'
```

Or if you don't want the activity to respond to cancelation:

```ts
export { setTimeout as delay } from 'timers/promises'
```